middleware/lib/claim-client.js

```diff
@@ -49,13 +60,32 @@ class ClaimClient {
     return this._signer;
   }
 
+  /**
+   * Generates a JSON timestamp in UTC time.
+   */
+  get timestamp() {
+    const d = new Date();
+    return new Date(
+      Date.UTC(
+        d.getUTCFullYear(),
+        d.getUTCMonth(),
+        d.getUTCDate(),
+        d.getUTCHours(),
+        d.getUTCMinutes(),
+        d.getUTCSeconds(),
+        d.getUTCMilliseconds(),
+      ),
+    ).toJSON();
+  }
+
   /**
    * Add 1 or more files to an existing claim.
    *
-   * @param {string} vin                     - The VIN of the claim.
-   * @param {Express.Request['files']} files - Array of uploaded files.
+   * @param {string} vin                                      - The VIN of the claim.
+   * @param {Express.Request['files']} files                  - Array of uploaded files.
+   * @param {keyof typeof Protos.File.Type} [fileType='NONE'] - The type of the files.
    */
-  async addFiles(vin, files) {
+  async addFiles(vin, files, fileType = 'NONE') {
```

Comment:
@ej3076 @sloan-seamus @dbartalino You'll need to work together to figure out how you guys wanna implement this in the middleware api and form.

Reply:
This can be handled purely in the frontend, middleware shouldn't need to double-confirm the file type

Reply:
You need to pass the filetype to the API though when uploading files.